### ISSUE TITLE
'Add Friend' method not working for PS5

### ISSUE THREAD

**Issue by KelvinLu93** [Pugmatt/BedrockConnect#523]
### What happened?

I've had 2 friends who play on console (1 on xbox, 1 on ps5) unable to get BCMain or any of its alts to appear on their friends list after they add it. I've tested both BCMain and BCMain1 and was able to get both to appear on my friends list (Windows 10). I thought the bots' friend lists might have been full so I removed them from my friends and had them try re-adding, this was not successful either.

### Expected Behaviour?

The bot accepts friend request from ps5 player, allowing them to join their game.

### Steps to reproduce.

1: Add BCMain/alts from console
2: Bot does not show up in friends list (Waited over 10 mins and tried subsequently resending request)

### Minecraft Bedrock Version

Latest release, 2.27.11

**Comment by LazyBirb** [comment]
Have you tried using one of the other hosted instances by the community?

**Comment by KelvinLu93** [comment]
Yes, they have tried several different ones. I didn’t ask which ones but I’ll try to find out

**Comment by Pugmatt** [comment]
BCMain's friendslist is currently full, so that won't probably won't currently work. BCMain1 was having some server issues a couple of days ago, so that may have been why. Should be working at this point. If not there are more listed under the ["Publicly available BedrockConnect instances" section](https://github.com/Pugmatt/BedrockConnect?tab=readme-ov-file#publicly-available-bedrockconnect-instances) as LazyBirb mentioned.

**Comment by ujex256** [comment]
https://github.com/rtm516/MCXboxBroadcast/issues/122

It seems that mojang made changes in 1.21.2 and the author of MCXboxBroadcast is currently working on a fix.

**Comment by LazyBirb** [comment]
That seems odd, I havent seen any problems with MCXboxBroadcast on my two instances after the update

**Comment by LazyBirb** [comment]
Nevermind i take it back, i forgot to update my client

**Comment by KelvinLu93** [comment]
Update: one friend says BCMain was added, but cannot be successfully joined. Instead, it shows this message when trying:

![image](https://github.com/user-attachments/assets/1ced5729-c057-4d45-8a4d-16b5c5fa557e)

**Comment by LazyBirb** [comment]
> Update: one friend says BCMain was added, but cannot be successfully joined. Instead, it shows this message when trying:
> 
> ![image](https://private-user-images.githubusercontent.com/24474186/358974673-1ced5729-c057-4d45-8a4d-16b5c5fa557e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQwOTE4MTgsIm5iZiI6MTcyNDA5MTUxOCwicGF0aCI6Ii8yNDQ3NDE4Ni8zNTg5NzQ2NzMtMWNlZDU3MjktYzA1Ny00ZDQ1LThhNGQtMTZiNWM1ZmE1NTdlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODE5VDE4MTgzOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI4ZjJlMWYyNmMwZDc3Y2RkYzBhNjJmNzI5YjAxOTc5MTkxZDdlZmMzMjNhMmY0ZjRiZDhlYzgwNWYwYjlkZmQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.K8vd1psCH82iL-3wf5pL6-E2omZwglZowbLNvoSWTjg)

Thats due to the mentioned bug in MCXboxBroadcast https://github.com/rtm516/MCXboxBroadcast/issues/122

**Comment by CoveWolf** [comment]
The dev rtm516 of MCXboxBroadcaster and Tim203 fixed it! They have released a dev build that seems to be working! Though at this point you can only get access by donating. I have done so and am now hosting a bedrock connect instance using MCXboxBroadcaster! I don't want to self-promote but if you add "CoveWBC3" it should work for you. Though note it may take a bit to load and it's still a dev build, so no promises it will consistently work.

**Comment by Pugmatt** [comment]
With the public release of the fixed MCXboxBroadcaster version today, BCMain and BCMain1 should now be working again as of today. Note you will need to re-add one of them to begin using it again (As I've re-created the accounts to start the friendlists fresh)

**Comment by LazyBirb** [comment]
As of right now I'm bringing my instance back up running with MCXboxBroadcast, running as usual with the normal restart schedule
